code/game/machinery/autolathe.dm

```diff
@@ -96,7 +96,7 @@
 	SIGNAL_HANDLER
 
 	//we use initial(active_power_usage) because higher tier parts will have higher active usage but we have no benifit from it
-	if(directly_use_power(ROUND_UP((amount_inserted / (MAX_STACK_SIZE * SHEET_MATERIAL_AMOUNT)) * 0.02 * initial(active_power_usage))))
+	if(directly_use_energy(ROUND_UP((amount_inserted / (MAX_STACK_SIZE * SHEET_MATERIAL_AMOUNT)) * 0.02 * initial(active_power_usage))))
```

Comment:
```suggestion
	if(directly_use_energy(ROUND_UP((amount_inserted / (MAX_STACK_SIZE * SHEET_MATERIAL_AMOUNT)) * 0.02 * initial(active_power_usage) KILO JOULES)))
```

Because this proc uses the apc cell directly & since all cells standard capacity has been scaled up 1e3(or 1 kilo joules) we need to compensate for that

Reply:
I'll just scale the active_power_usage instead, since we don't need to double apply units. On the topic of active_power_usage, I've seen it both for time based processing and action based. I guess I could make a comment saying the unit is ambiguous.